README.md

```diff
@@ -68,6 +68,15 @@ py systemlinkmigrate.py --capture --tag --dir="C:\migrate_8-3-2020
 ````
 These arguments will capture tag and tag history data and store them in the directory `C:\migrate_8-3-2020`
 
+## Migrating tag history to nitaghistorian database
+Due to a bug introduced in SystemLink 2020R2 it is possible for tag history data to be stored in the incorrect database within the MongoDB instance. **This bug only affects environments where a remote MongoDB server is used.** To determine if you are in this state connect a MongoShell or visual database tool such as [Mongo Compase](https://www.mongodb.com/products/compass) to your Mongo instance and check if the collections `metadata` and `values` are in the default authentication database (typically the `admin` database).
```

Comment:
Tag history is the only one that is impacted when upgrading from R1, but is at least 1 other R2 service (security) that's also storing data in the wrong database.  Should the docs / param name cover all of it?

Might also be worth noting that a configuration change is necessary as well